packages/terra-filter-view/src/terra-dev-site/doc/example/DefaultFilterView.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import FilterView from 'terra-filter-view/lib/FilterView';
+
+const FilterViewDefault = () => (
+  <FilterView>
+    <FilterView.Option value="red" display="Red" key="red" />
+    <FilterView.Option value="blue" display="Blue" key="blue" />
+    <FilterView.Option value="green" display="Green" key="green" />
+  </FilterView>
+);
+
+export default () => FilterViewDefault();
```

Comment:
Is there a reason we're exporting a function to call the `FilterViewDefault` function?

Reply:
I'll change this to JSX instead to stay consistent.